acstools/acszpt.py

```diff
@@ -54,76 +56,69 @@
        Angstrom erg / (Angstrom cm2 s) mag(ST)   mag   mag(AB)
 ------ -------- ---------------------- ------- ------- -------
  F435W   4329.2              3.148e-19  25.155  25.763  25.665
+
 """
+from six.moves.urllib.request import urlopen
+from six.moves.urllib.error import URLError
 
-from collections import OrderedDict
 import datetime as dt
 import logging
-from urllib import request, error as request_errors
+from collections import OrderedDict
 
-from astropy.table import Table
 import astropy.units as u
+from astropy.table import Table
 from bs4 import BeautifulSoup
 
 __taskname__ = "acszpt"
-__version__ = "1.0"
 __author__ = "Nathan Miles"
+__version__ = "1.0"
 __vdate__ = "22-Jan-2019"
+
 __all__ = ['Query']
 
 # Initialize the logger
 logging.basicConfig()
-LOG = logging.getLogger('{}.{}'.format(__taskname__,__all__[0]))
+LOG = logging.getLogger('{}.{}'.format(__taskname__, 'Query'))
 LOG.setLevel(logging.INFO)
 
 
 class Query(object):
-    """ Class used to interface with the ACS Zeropoints Calculator API.
+    """Class used to interface with the ACS Zeropoints Calculator API.
 
     Parameters
     ----------
     date : str
         Input date in the following ISO format, YYYY-MM-DD.
-    detector : str
+
+    detector : {'HRC', 'SBC', 'WFC'}
         One of the three channels on ACS: HRC, SBC, or WFC.
-    filt : str, optional
+
+    filt : str or `None`, optional
         One of valid filters for the chosen detector. If no filter is supplied,
-        all of the filters for the chosen detector will be used.
+        all of the filters for the chosen detector will be used:
 
             * HRC:
                  F220W, F250W, F330W,
                  F344N, F435W, F475W,
                  F502N, F550M, F555W,
                  F606W, F625W, F658N, F660N,
                  F775W, F814W, F850LP, F892N
-
             * WFC:
                  F435W, F475W,
                  F502N, F550M, F555W,
                  F606W, F625W, F658N, F660N,
                  F775W, F814W, F850LP, F892N
             * SBC:
```

Comment:
This invoked the http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists rendering. FYI.